doge-webapp/build.gradle

```diff
@@ -1,23 +1,30 @@
 description = "doge webapp"
-
 apply plugin: 'war'
-apply plugin: 'org.springframework.boot'
 
-buildscript {
-    ext {
-        springBootVersion = '1.5.1.RELEASE'
-    }
-    repositories {
-        mavenCentral()
-    }
-    dependencies {
-        classpath("org.springframework.boot:spring-boot-gradle-plugin:${springBootVersion}")
-    }
+repositories {
+    mavenCentral()
+}
+dependencies {
+    testCompile('junit:junit:4.12')
 }
 
+apply plugin: 'org.springframework.boot'
+buildscript {
+    repositories { mavenCentral() }
+    dependencies { classpath("org.springframework.boot:spring-boot-gradle-plugin:1.5.1.RELEASE") }
+}
 dependencies {
     compile('org.springframework.boot:spring-boot-starter-thymeleaf')
     compile('org.springframework.boot:spring-boot-starter-web')
     testCompile('org.springframework.boot:spring-boot-starter-test')
-    testCompile('junit:junit:4.12')
 }
+bootRun {
+    addResources = true
+}
+
+apply plugin: 'com.moowork.node'
+buildscript {
+    repositories { maven { url "https://plugins.gradle.org/m2/" } }
+    dependencies { classpath("com.moowork.gradle:gradle-node-plugin:1.1.1") }
```

Comment:
I separated them to denote why I was using this repo / dependency.  (If you look at the file without the diff, you can see that the node stuff is grouped and the spring boot stuff is grouped, since gradle closures don't care about being separate.)  I guess that wasn't obvious though.  Opinion?